nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/security/LockSecurityPolicy.java

```diff
@@ -37,10 +37,15 @@
  */
 public class LockSecurityPolicy extends AbstractSecurityPolicy {
 
+    protected static ThreadLocal<Boolean> ignorePolicy = new ThreadLocal<>();
+
     @Override
     public Access checkPermission(Document doc, ACP mergedAcp, NuxeoPrincipal principal, String permission,
             String[] resolvedPermissions, String[] additionalPrincipals) {
         Access access = Access.UNKNOWN;
+        if (Boolean.TRUE.equals(ignorePolicy.get())) {
```

Comment:
Move `Boolean.TRUE.equals(ignorePolicy.get())` to a static method called `isIgnorePolicy`